instrumentation/net/http/otelhttp/test/handler_test.go

```diff
@@ -528,3 +528,43 @@ func TestWithRouteTag(t *testing.T) {
 		}
 	}
 }
+
+func BenchmarkHandlerServeHTTP(b *testing.B) {
+	tp := sdktrace.NewTracerProvider()
+	mp := sdkmetric.NewMeterProvider()
+
+	r, err := http.NewRequest(http.MethodGet, "http://localhost/", nil)
```

Comment:
Is it possible to structure this so the "extra" attributes are also set?
"net.host.port", "net.sock.peer.addr", "net.sock.peer.port", "user_agent.original", "http.client_ip".

Reply:
I can, but there's already a benchmark in the semconv package, which I believe would be better to test specific behavior.
https://github.com/open-telemetry/opentelemetry-go-contrib/blob/fa9fc0cf0b33211ff66d836e6636aa98b9e6f293/instrumentation/net/http/otelhttp/internal/semconv/bench_test.go#L19

Reply:
Thats fine.  I'm hoping this benchmark can be somewhat of a canonical benchmark we can use to evaluate the impact of changes to the SDKs as well.  So as long as this is reasonably representative of instrumentation, its fine

Reply:
Yeah, but they only give a high-level overview, and a performance change could be hidden by something else.
They shouldn't replace lower-level benchmarks